### DISCUSSION TITLE
Node ID mapping when reading ABAQUS mesh

### DISCUSSION THREAD

**Discussion by yw5aj** [libMesh/libmesh#3605] [Q&A] [answered]
Hi all,

I noticed that when I read in an ABAQUS mesh, the node mapping wasn't a simple +1 offset (as I imagined from https://github.com/libMesh/libmesh/blob/fd900b54a02b9d1b99f0a4371d4acb4761e57b0f/src/mesh/abaqus_io.C#L532). I am trying to read some nodal attributes from my code and therefore would need a mapping from the ABAQUS nodal ID to the libmesh nodal ID. Could you help share a pointer? I can share an example below too which will illustrate the case. Thank you!

[ReadShell.zip](https://github.com/libMesh/libmesh/files/12166521/ReadShell.zip)

Shawn

**Comment by roystgnr** [comment, marked as answer]
Does the Abaqus file have its nodes written non-contiguously?  Have you left mesh renumbering enabled on a `DistributedMesh`?  Those are the only things I can think of that would give you unexpected numberings.

**Reply by yw5aj** [reply]
Renumbering! After doing `mesh.allow_renumbering(false);` it was all fixed. Sorry about the stupid mistake and thanks for the pointer!
